lib/mix/tasks/grf.server.ex

```diff
@@ -20,8 +20,10 @@ defmodule Mix.Tasks.Grf.Server do
       $ mix do deps.loadpaths --no-deps-check, phx.server
   """
 
-  @impl true
+  @impl Mix.Task
   def run(args) do
+    port = GriffinSSGApp.http_port
+    IO.puts "Starting webserver on #{port}"
```

Comment:
It wasn't obvious to me what's the default port (it has changed since `0.2.0` version) so I decided to print it to the console :).

Reply:
In the rest of the app we're using [`Mix.shell().info/1`](https://hexdocs.pm/mix/1.12.3/Mix.Shell.html#c:info/1) to write to the console, could we use the same function here?

Since we're adding this, what do you think about printing a URL like `http://localhost:#{port}` so that users with modern terminals can Ctrl + click to navigate directly to it?

Reply:
Sure, no problem!